src/widgets/__tests__/index.test.ts

```diff
@@ -121,6 +121,13 @@ function initiateAllWidgets(): Array<[WidgetNames, Widget]> {
           attributes: ['attr1', 'attr2'],
         });
       }
+      case 'EXPERIMENTAL_dynamicWidgets': {
+        const EXPERIMENTAL_dynamicWidgets = widget as Widgets['EXPERIMENTAL_dynamicWidgets'];
+        return EXPERIMENTAL_dynamicWidgets({
+          container,
```

Comment:
```suggestion
          container,
          transformItems() {
            return [];
          }
```

as it's now required